test/e2e/go.mod

```diff
@@ -0,0 +1,39 @@
+module github.com/operator-framework/operator-lifecycle-manager/test/e2e
+
+go 1.14
+
+require (
+	github.com/blang/semver v3.5.1+incompatible
```

Comment:
Does the e2e module really need all of these deps?

Reply:
Sadly yes - but helm recently shipped a PR that supports 1.20 so I decided to avoid creating a new module as we hit other dependency issues. We should however investigate how we can reduce our e2e tests on code implemented in the OLM project.